src/psyclone/gocean1p0.py

```diff
@@ -129,8 +129,6 @@ def gen(self):
         # include the field_mod module
         psy_module.add(UseGen(psy_module, name="field_mod"))
```

Comment:
Not your fault but pylint complains about white space at line 71

Reply:
Fixed